lib/beaker-pe/install/pe_utils.rb

```diff
@@ -1087,6 +1098,9 @@ def register_feature_flags!(opts)
         # @param [Hash] opts The Beaker options hash
         # @return [BeakerAnswers::Answers] the generated answers object
         def generate_installer_conf_file_for(host, hosts, opts)
+          possible_mco_enabled_setting = get_mco_setting(hosts)
+          opts ||= {}
+          opts = possible_mco_enabled_setting.deep_merge(opts)
```

Comment:
With the change above, we could've just merged the contents of answers without a deep merge.